template/webpack.config.js

```diff
@@ -14,6 +14,9 @@ require('./prepare')();
 // Generate platform-specific webpack configuration
 const config = (platform, launchArgs) => {
 
+  const command = launchArgs.split(' ')[0];
+  const isDevelopment = command === 'debug'
```

Comment:
Isn't _isDevelopment_ confusing? You are checking if we're running debug, not really development. If you want to cover development as well, then maybe you can do:
`const isDevelopment = command !== "build";`

What do you think?